content/en/docs/tasks/administer-cluster/subpath-env.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Using subPath with expanded environment variables 
+content_type: task
+---
```

Comment:
```suggestion
min-kubernetes-server-version: v1.17
---
```
(is that right?)

Reply:
I can find it in 1.15, so I'll use that.